lbrynet/extras/wallet/dewies.py

```diff
@@ -30,4 +30,4 @@ def lbc_to_dewies(lbc: str) -> int:
 
 
 def dewies_to_lbc(dewies) -> str:
-    return satoshis_to_coins(dewies)
+    return dewies if isinstance(dewies, str) else satoshis_to_coins(dewies)
```

Comment:
I'm skeptical about this change. This was needed because the claim was re-processed. In which case dewies would already be in string(desired) format.